### DISCUSSION TITLE
EJBCA adminweb 404 after upgrading to 8.2.0.1

### DISCUSSION THREAD

**Discussion by johnlanham1957** [Keyfactor/ejbca-ce#704] [General]
Hi Folks, After what looks like a successful upgrade from 7.11.0 to 8.2.0.1 I am getting a 404 error when trying to access adminweb, I see in release notes that this appears to be deprecated, is this accurate and what URL should I now be using, I dont see anything in the docs??

Thanks

John

**Comment by primetomas** [comment]
I guess you mean the publicweb, that is deprecated in favor of the RA Web. Access the adminweb, and there is a link to RA web.

**Reply by johnlanham1957** [reply]
Yes, accessing the following URL's result in a 404 error

https://172.16.8.123:8443/ejbca/ra

https://172.16.8.123:8443/ejbca/adminweb

**Reply by primetomas** [reply]
Then I don't think your upgrade went well and EJBCA is not deployed. Check server.log for errors.

**Reply by johnlanham1957** [reply]
Agreed, I just found this in my log, JDK 17 is installed and set as the default JDK!!

Failed to link org/ejbca/ui/web/admin/cainterface/CACertServlet (Module \"deployment.ejbca.ear.adminweb.war:main\" from Service Module Loader): org/ejbca/ui/web/admin/cainterface/CACertServlet has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 52.0"
